src/dashboard/app.py

```
@@ -5,80 +5,259 @@
+def load_dashboard_data():
+    """results 디렉토리의 분석 결과를 로드하거나, 없을 경우 기본 분석 수행"""
+    
+    # 1. 코호트 리텐션 데이터
+    cohort_path = RESULTS_DIR / "cohort_retention.csv"
+    if cohort_path.exists():
+        cohort_df = pd.read_csv(cohort_path)
+    else:
+        try:
+            customers, events = load_data(DATA_DIR)
+            cohort_df = build_cohort_retention(customers, events)
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)